service-base/src/main/java/org/eclipse/hono/service/http/DefaultFailureHandler.java

```diff
@@ -39,6 +39,7 @@
  */
 public class DefaultFailureHandler implements Handler<RoutingContext> {
 
+    private static final String MSG_NO_ERROR = "<no error>";
```

Comment:
I would rather send an empty string in this case. Getting a response with an error code that contains a body saying "no error" would be quite confusing, wouldn't it?

Reply:
Maybe "unknown error" is better. But having nothing at all again makes it hard to figure out what went wrong. Having some kind of string helps you find the right spot in the source code easily.

Reply:
How does this help?